# Service to add the requested folder and all subfolders to MATLAB's path

string folder       # folder to add to MATLAB's path
---
int64 ret         # Set to 1 (success) or 0 (error)
string msg        # Error description
